tensorflow/compiler/mlir/tensorflow/ir/tf_traits.h

```diff
@@ -133,6 +133,10 @@ class CwiseBinary : public TraitBase<ConcreteType, CwiseBinary> {};
 template <typename ConcreteType>
 class CwiseUnary : public TraitBase<ConcreteType, CwiseUnary> {};
 
+// Unary element wise function that is a self inverse, example tf.Negate operation
+template <typename ConcreteType>
+class Involution : public TraitBase<ConcreteType, Involution> {};
```

Comment:
Can `verifyTrait` be implemented, checking if the op has one operand and one result?